eval/eval_test.go

```diff
@@ -236,18 +238,24 @@ func TestEval(t *testing.T) {
 				require.NoError(t, err)
 				sortEnvironmentDiagnostics(loadDiags)
 
-				_, checkDiags := CheckEnvironment(context.Background(), e.Name(), env, testProviders{}, &testEnvironments{basePath})
+				check, checkDiags := CheckEnvironment(context.Background(), e.Name(), env, testProviders{}, &testEnvironments{basePath})
 				sortEnvironmentDiagnostics(checkDiags)
 
 				actual, evalDiags := EvalEnvironment(context.Background(), e.Name(), env, rot128{}, testProviders{}, &testEnvironments{basePath})
 				sortEnvironmentDiagnostics(evalDiags)
 
+				evalJSON := esc.NewValue(actual.Properties).ToJSON(true)
+				checkJSON := esc.NewValue(check.Properties).ToJSON(true)
+
 				bytes, err := json.MarshalIndent(expectedData{
 					LoadDiags:   loadDiags,
 					CheckDiags:  checkDiags,
 					EvalDiags:   evalDiags,
 					Environment: actual,
+					EvalJSON:    evalJSON,
+					CheckJSON:   checkJSON,
 				}, "", "    ")
+				bytes = append(bytes, '\n')
```

Comment:
just curious why we need this bit

Reply:
coz I hate this icon
<img width="723" alt="Screenshot 2023-11-07 at 1 51 00 PM" src="https://github.com/pulumi/esc/assets/24326455/e85fec73-4065-400a-b09b-1cae6eb9582d">